README.old.md

```diff
@@ -0,0 +1,2 @@
+# math-magicians
```

Comment:
- [ ] [OPTIONAL] There are two readme files. Please delete one of those to make your project clean.